src/Functional.js

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+
+const todoList = [
+    'Do React course',
+    'Go Shopping',
+    'Read',
+];
+
+function Functional() {
+    const [todo, setTodo] = useState(false);
```

Comment:
As default value, it's more semantically correct to just leave it if there's no initial value

Reply:
I mean 
```
    const [todo, setTodo] = useState();
```